domain/src/main/java/org/oppia/android/domain/oppialogger/analytics/PerformanceMetricsLogger.kt

```diff
@@ -15,71 +21,109 @@ class PerformanceMetricsLogger @Inject constructor(
 ) : ApplicationStartupListener {
 
   /**
-   * Use a large Long value such that the time difference based on any timestamp will be negative
-   * and thus ignored until the app records initial time during [onCreate].
+   * Timestamp indicating the time of application start-up. It will be used to calculate the
+   * cold-startup latency of the application.
+   *
+   * We're using a large Long value such that the time difference based on any timestamp will be
+   * negative and thus ignored until the app records initial time during [onCreate].
    */
   private var firstTimestamp: Long = Long.MAX_VALUE
 
   override fun onCreate() {
     firstTimestamp = oppiaClock.getCurrentTimeMs()
   }
 
-  /** Logs the apk size of the application as a performance metric. */
-  fun logApkSize() {
+  /**
+   * Logs the apk size of the application as a performance metric for the current state of the app.
+   * It must be noted that the value of this metric won't change across calls during the same
+   * session.
+   *
+   * @param [currentScreen] denotes the application screen at which this metric has been logged.
```

Comment:
```suggestion
   * @param currentScreen denotes the application screen at which this metric has been logged
```

Nit: at-clauses don't end with punctuation unless they're more than one sentence. Ditto elsewhere in the PR.

Also, the parameter name shouldn't include a link reference for the first part of the at-clause. Ditto also elsewhere in the PR.

Reply:
done.